rocketpy/simulation/flight.py

```diff
@@ -1006,9 +1005,8 @@ def __init__(
                                     )
                                     # Gets height above ground level considering noise
                                     hAGL = (
-                                        self.env.pressure.find_input(
+                                        self.env.barometric_height(
                                             pressure + noise,
```

Comment:
Remove the leading comma and run black again

```suggestion
                                            pressure + noise
```